modules/exploits/multi/http/atutor_sqli.rb

```diff
@@ -243,7 +243,7 @@ def get_ascii_value(sql, cookie)
        mid = (lower + upper) / 2
        sqli = "#{sql}>#{mid}"
        result = perform_request(sqli, cookie)
-       if result =~ /There are \d entries./
+       if result =~ /There are \d{0,8} entries\./
```

Comment:
Right. But if you do ```\d{0,8}```, that can match a null value, whereas ```\d{1,8}``` will match a number 1-8 digits long.